frontend/src/components/overviews/OverviewWithMidline.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+	<div class="table">
+		<div class="line"></div>
+		<div v-for="el in dataArray" class="table_row">
+			<div class="table_cell table_left">{{ el.left }}:</div>
+			<div class="table_cell table_right"><i>{{ el.right }}</i></div>
+		</div>
+	</div>
+</template>
+
+<script>
+export default {
+name: 'OverviewWithMidline',
+	props: {
+		dataArray: Array,
+	},
```

Comment:
If you create a type/interface for this in the src/types directory you can import the type in both this file and the file that uses this component. 
You can then define the type of the array like so

```vue
<script lang="ts">
import { defineComponent } from 'vue';
import type { typename_you_pick_for_this_type } from '../../types/type_filename_you_make_for_this_type';

export default defineComponent({
	name: 'OverviewWithMidline',
	props: {
		dataArray: Array<typename_you_pick_for_this_typee>,
	},
});
</script>
```